lxc/image.go

```diff
@@ -109,17 +109,29 @@ hash or alias name (if one is set).`))
 	return cmd
 }
 
-func (c *cmdImage) dereferenceAlias(d lxd.ImageServer, imageType string, inName string) string {
+func (c *cmdImage) dereferenceAlias(d lxd.ImageServer, imageType string, inName string) (*api.Image, string, error) {
```

Comment:
a docblock comment here would be useful to the caller to allow them to understand what the 2nd return value is.

Reply:
fixed